Scripts/travis-build-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -x -o pipefail
+
+echo "Building"
+
+rm -rf ${HOME}/Library/Developer/Xcode/DerivedData/*
+
+# -jobs -- specify the number of concurrent jobs
+# `sysctl -n hw.ncpu` -- fetch number of 'logical' cores in macOS machine
+xcodebuild -jobs `sysctl -n hw.ncpu` test -workspace Contentful.xcworkspace -scheme Contentful \
+  -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 6s,OS=9.3" \
```

Comment:
Just realized i could set up a matrix for the OS version though which might be useful at @dlitvakb . Will do this in the future